targets/TARGET_NUVOTON/TARGET_M451/TARGET_NUMAKER_PFM_M453/device.h

```diff
@@ -38,7 +38,7 @@
 #define DEVICE_SPI_ASYNCH       1
 #define DEVICE_SPISLAVE         1
 
-#define DEVICE_CAN              0
+//#define DEVICE_CAN              1
```

Comment:
Define in device.h is legacy, so I follow most platforms to define CAN in "device_has" field of targets.json .